### ISSUE TITLE
select2 country and state style issues when expanding upward

### ISSUE THREAD

**Issue by seanmcintyre** [sharewave/frontus#1617]
![screen shot 2015-02-02 at 2 47 29 pm](https://cloud.githubusercontent.com/assets/319302/6007743/80c7ba9a-aaea-11e4-86a9-65d1c9b17889.png)

**Comment by RWalport** [comment]
This is true of all the select2s so definitely worth looking into (though yes also definitely low priority)
